advanced/events.md

```diff
@@ -130,13 +130,13 @@ tinymce.init({
 
 ### SetContent
 
-Gets fired after contents been inserted into the editor.
+Gets fired after the content has been inserted into the editor.
 
 #### Parameters
 * **content** `String` - The HTML content that was inserted into the editor.
 * **selection** `Boolean` - True/False if the content was inserted at selection or replaced all contents.
 
-Here is an example on log the content that got inserted into the editor.
+Here is an example of a content log that got inserted into the editor.
```

Comment:
Here is a console log example when content has been inserted into the editor

Reply:
In the following example, the content is logged to the console in response to he SetContent event.

Reply:
To keep the wording consistent with the other examples on this page, I have changed it to:
"Here is an example of the content logged to the console in response to the SetContent event."